README.md

```diff
@@ -111,7 +111,7 @@ $ mvn verify  -Pspotbugs
 
 ```bash
 # At the root of the project
```

Comment:
Not at the root of the project. Maybe "# in the docs directory"

Reply:
Or `mvn -f docs/pom.xml package -Pjavadoc`